miner.py

```diff
@@ -54,8 +54,8 @@ def mine():
 
     attempts = 0
 
-    keep = secrets.token_hex(32)
-    subsidy = secrets.token_hex(32)
+    keep = generate_new_secret(webcash_wallet, chain_code="MINING")
+    subsidy = generate_new_secret(webcash_wallet, chain_code="PAY")
```

Comment:
This isn't resolved, but fixing it might be a bit more complicated than is warranted for this PR. I'm marking my review as 'approved' in case you want to handle this separately.

Reply:
How about this? 7a62404c178d3335c7bc1a973b9c955a54994a82

Reply:
Looks good, although I didn't test it.

Reply:
> On the other hand, that would make mining deterministic, such that if you re-start the miner it will start looking at the same work again. We really ought to put a timestamp in the mining payload structure.

67737d33bdd57ca0c19a2e8ef25585fb940b6a8e

Reply:
I’ll update webminer to do the same thing.